src/services/localSSHService.ts

```diff
@@ -77,7 +77,7 @@ export class LocalSSHService extends Disposable implements ILocalSSHService {
         return {
             Host: '*.' + getLocalSSHDomain(host),
             StrictHostKeyChecking: 'no',
-            ProxyCommand: `"${launcher}" "${process.execPath}" "${proxyScript}" --ms-enable-electron-run-as-node %h ${extIpcPort} "${Configuration.getLocalSSHLogPath()}"`
+            ProxyCommand: `"${launcher}" "${process.execPath}" "${proxyScript}" --ms-enable-electron-run-as-node %h ${extIpcPort} %r`
```

Comment:
username is not needed, we can get it from `onAuthenticating`, I will revert this part